### PR TITLE
Added Navbar in login Page

### LINKED ISSUE #338: Add navbar in About and Login page
## 🚀 Feature

Hey @nilisha-jais @niloysikdar 
I would like to add navbar in about and login page
Please assign it to me.
Thanks

## Motivation

(Please outline the motivation for the proposal.)

## Pitch

(Please explain why this feature should be implemented and how it would be used.)

## Add relevant screenshot or video (if any)

(Add here)

### PR DESCRIPTION
**Pull request by Shaishav0507**
### Description

Fixes: #338 
Hey @niloysikdar 
Added Navbar in login page.
Please check and update.
Thanks

### Checklist

- [ ] I am making a proper pull request, not a spam.
- [ ] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

(Write your answer here.)

## Add relevant screenshot or video (if any)

(Add here)

### CODE REVIEW

File: login/login.html [hunk 1]
```diff
@@ -6,13 +6,61 @@
...
+        <ul class="navbar-nav">
+          <!--  me-auto mb-2 mb-lg-0 -->
+          <li class="nav-item">
+            <a class="nav-link current" aria-current="page" href="../index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link current" href="./About/index.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link current" href="../Contact-Us/contact.html">Contact Us</a>
```
Comment (niloysikdar) [inline login/login.html:53]: Check this href link

**Review by niloysikdar — CHANGES_REQUESTED** [review]
The About link isn't working, please fix it. @Shaishav0507

File: login/login.html (hunk 1 above)
Comment (Shaishav0507) [reply login/login.html:53]: Ok I'll do

Comment (Shaishav0507) [reply login/login.html:53]: Means it is not redirecting to that page right

Comment (niloysikdar) [reply login/login.html:53]: Yes

**Comment by Shaishav0507** [comment]
Ok I'll do

**Comment by Shaishav0507** [comment]
@niloysikdar Done please check.

**Review by niloysikdar — APPROVED** [review]
Working @Shaishav0507 , thanks.

**Pull request merged**